configs/regenerate.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
-go run ./cmd/acra-server/*.go --dump_config
-go run ./cmd/acra-connector/*.go --dump_config
-go run ./cmd/acra-translator/*.go --dump_config
-go run ./cmd/acra-addzone/*.go --dump_config
-go run ./cmd/acra-webconfig/*.go --dump_config
-go run ./cmd/acra-rollback/*.go --dump_config
-go run ./cmd/acra-keymaker/*.go --dump_config
-go run ./cmd/acra-poisonrecordmaker/*.go --dump_config
-go run ./cmd/acra-authmanager/*.go --dump_config
-go run ./cmd/acra-rotate/*.go --dump_config
+
+binaries=(server connector translator addzone webconfig rollback keymaker poisonrecordmaker authmanager rotate)
+
+args="--dump_config"
+
+
+for cmd in "${binaries[@]}"; do
+ if [[ "$#" == "1" ]]; then
+    args="--dump_config --config_file=$1/acra-${cmd}.yaml"
+ fi
+ go run ./cmd/acra-${cmd}/*.go `echo ${args}`
```

Comment:
Why use command substitution here instead of simply splicing the arguments?
```suggestion
 go run ./cmd/acra-${cmd}/*.go ${args}
```

Reply:
because this script do only one thing: generate all configs and doesn't require users to pass any args. plus script uses only args that support all binaries from `cmd/*`. it's general parameters like '--help'/'--config_filg'/'--dump_config'. all other parameters may be differ between all binaries